frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -57,7 +75,7 @@ export const cleanFunnelParams = (filters: FilterType): FilterType => {
 const isStepsEmpty = (filters: FilterType): boolean =>
     [...(filters.actions || []), ...(filters.events || [])].length === 0
 
-export const funnelLogic = kea<funnelLogicType>({
+export const funnelLogic = kea<funnelLogicType<TimeStepOption>>({
```

Comment:
I think the new kea typegen no longer requires injecting deps?

Reply:
hmm I think my file just added this by itself, I didn't manually put this in, should I still take it out or is this part of kea magic? :o